### PR TITLE
Dev/fixup plugin bundle

### PR DESCRIPTION
**Pull request by linus-sherrill**
Added support for including plugins and dependent libraries into the plugin fixup process.

### CODE REVIEW

File: CMake/utils/maptk-utils-targets.cmake [hunk 1]
```diff
@@ -37,6 +37,11 @@ define_property(GLOBAL PROPERTY maptk_plugin_libraries
   FULL_DOCS "List of generated static/shared plugin module libraries"
   )
 
+define_property(GLOBAL PROPERTY maptk_bundle_paths
+  BRIEF_DOCS "Paths neede by fixup_bindle"
```
Comment (Purg) [inline CMake/utils/maptk-utils-targets.cmake]: "neede" -> "needed"

**Comment by linus-sherrill** [comment]
Updated documentation in ```maptk-utile-targets.cmake``` - corrected typo's and spelling problems.

File: CMake/utils/maptk-utils-targets.cmake [hunk 2]
```diff
@@ -37,6 +37,11 @@ define_property(GLOBAL PROPERTY maptk_plugin_libraries
   FULL_DOCS "List of generated static/shared plugin module libraries"
   )
 
+define_property(GLOBAL PROPERTY maptk_bundle_paths
+  BRIEF_DOCS "Paths needed by fixup_bindle"
```
Comment (mleotta) [inline CMake/utils/maptk-utils-targets.cmake]: I just spotted a typo: bindle --> bundle

**Comment by mleotta** [comment]
Fixes #125

**Pull request merged**